### PR TITLE
Adding aruba_command module along with unit tests.

### PR DESCRIPTION
**Pull request by jmighion**
##### SUMMARY

New module `aruba_command`.

##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

aruba_command
##### ANSIBLE VERSION

```
$ ansible --version
ansible 2.4.0 (devel 10233ef3b5) last updated 2017/07/06 18:23:01 (GMT +000)
  config file = /home/jmighion/git/blank-ansible-repo/ansible.cfg
  configured module search path = [u'/home/jmighion/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/jmighion/git/ansible/lib/ansible
  executable location = /home/jmighion/git/ansible/bin/ansible
  python version = 2.7.5 (default, Aug  2 2016, 04:20:16) [GCC 4.8.5 20150623 (Red Hat 4.8.5-4)]
```

### CODE REVIEW

File: lib/ansible/module_utils/aruba.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+aruba_argument_spec = {
+    'host': dict(),
+    'port': dict(type='int'),
+    'username': dict(fallback=(env_fallback, ['ANSIBLE_NET_USERNAME'])),
+    'password': dict(fallback=(env_fallback, ['ANSIBLE_NET_PASSWORD']), no_log=True),
+    'ssh_keyfile': dict(fallback=(env_fallback, ['ANSIBLE_NET_SSH_KEYFILE']), type='path'),
+    'authorize': dict(fallback=(env_fallback, ['ANSIBLE_NET_AUTHORIZE']), type='bool'),
+    'auth_pass': dict(fallback=(env_fallback, ['ANSIBLE_NET_AUTH_PASS']), no_log=True),
+    'timeout': dict(type='int'),
```
Comment (ganeshrn) [inline lib/ansible/module_utils/aruba.py]: Where is the default timeout value set?

Comment (jmighion) [reply lib/ansible/module_utils/aruba.py]: Ah good catch. Looks like IOS has the same problem since this is mostly a copy & rewrite of IOS.

**Comment by jmighion** [comment]
Shippable failed building out rhel7.3 env. Closing and reopening to restart shippable.

**Comment by jmighion** [comment]
More envs timing out in shippable. Restarting.

File: lib/ansible/module_utils/aruba.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+aruba_argument_spec = {
+    'host': dict(),
+    'port': dict(type='int'),
+    'username': dict(fallback=(env_fallback, ['ANSIBLE_NET_USERNAME'])),
+    'password': dict(fallback=(env_fallback, ['ANSIBLE_NET_PASSWORD']), no_log=True),
+    'ssh_keyfile': dict(fallback=(env_fallback, ['ANSIBLE_NET_SSH_KEYFILE']), type='path'),
+    'authorize': dict(fallback=(env_fallback, ['ANSIBLE_NET_AUTHORIZE']), type='bool'),
+    'auth_pass': dict(fallback=(env_fallback, ['ANSIBLE_NET_AUTH_PASS']), no_log=True),
+    'timeout': dict(type='int', default=10),
```
Comment (ganeshrn) [inline lib/ansible/module_utils/aruba.py]: Setting default value here will trigger deprecation warning timeout. Please refer code here https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/junos.py#L52-L55 and
https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/junos.py#L69-L74 to work around it.

**Comment by jmighion** [comment]
Shippable timeout error. Restarting.

**Comment by jmighion** [comment]
Shippable still unstable. Restarting.

**Comment by jmighion** [comment]
Shippable unstable. Restarting.

**Review by ganeshrn — APPROVED** [review]

**Comment by jmighion** [comment]
Shippable failed building out an env. Closing and reopening to restart shippable.

**Pull request merged**